src/Symfony/Component/HttpKernel/DependencyInjection/RegisterControllerArgumentLocatorsPass.php

```diff
@@ -129,7 +127,7 @@ public function process(ContainerBuilder $container): void
                     /** @var \ReflectionParameter $p */
                     $type = preg_replace('/(^|[(|&])\\\\/', '\1', $target = ltrim(ProxyHelper::exportType($p) ?? '', '?'));
                     $invalidBehavior = ContainerInterface::IGNORE_ON_INVALID_REFERENCE;
-                    $autowireAttributes = $autowire ? $emptyAutowireAttributes : [];
+                    $autowireAttributes = null;
```

Comment:
The variable `$autowireAttributes` is used in 1 place when `$autowire` is `true`. But still need to be reset on each loop.